README.md

```diff
@@ -77,6 +77,14 @@ The following inputs can be used as `steps.with` keys:
 | `aws_create_vpc` | bool | `false` | Whether an AWS VPC should be created in the action. Otherwise, the existing default VPC will be used. |
 | `aws_extra_tags` | json | | A list of additional tags that will be included on created resources. Example: `{"key1": "value1", "key2": "value2"}`. |
 | `infrastructure_only` | bool | `false` | Set to true to provision infrastructure (with Terraform) but skip the app deployment (with ansible) |
+| **Domain and certificates configuration** |
+| `domain_name` | String | | Define the root domain name for the application. e.g. bitovi.com'. |
+| `sub_domain` | String | `${org}-${repo}-${branch}` | Define the sub-domain part of the URL. |
+| `root_domain` | Boolean | `false` | Deploy application to root domain. Will create root and www records. |
+| `cert_arn` | String | | Define the certificate ARN to use for the application. **See note**. |
+| `create_root_cert` | Boolean | `false`| Generates and manage the root cert for the application. **See note**.|
+| `create_sub_cert` | Boolean | `false` | Generates and manage the sub-domain certificate for the application. **See note**.|
+| `no_cert` | Boolean | `false` | Set this to true if no certificate is present for the domain. **See note**. |
```

Comment:
No. As a default config, if a domain name is passed, it will look up for an existing certificate. We could do change this to something like `use_existing_cert` kind of thing.